zombies-timer/src/main/java/org/phantazm/zombies/autosplits/config/ZombiesAutoSplitsConfig.java

```diff
@@ -0,0 +1,65 @@
+package org.phantazm.zombies.autosplits.config;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Objects;
+
+public record ZombiesAutoSplitsConfig(@NotNull String host, int port, boolean useLiveSplits, boolean useInternal) {
+
+    public static final String DEFAULT_HOST = "localhost";
```

Comment:
bit unsure as to what is being hosted here

Reply:
the mod can integrate with livesplits via the LiveSplit.Server Component